algorithms/src/std_algorithms/impl/Kokkos_InclusiveScan.hpp

```diff
@@ -101,9 +142,25 @@ struct InclusiveScanDefaultFunctor {
   }
 };
 
-//
-// exespace impl
-//
+// -------------------------------------------------------------
+// inclusive_scan_default_op_exespace_impl
+// -------------------------------------------------------------
+
+#if defined(KOKKOS_ENABLE_CUDA)
+template <class InputIteratorType, class OutputIteratorType>
+OutputIteratorType inclusive_scan_default_op_exespace_impl(
+    const std::string& label, const Cuda& ex, InputIteratorType first_from,
```

Comment:
The label is unused here. Maybe, add something like
```C++
 uint64_t kpID = 0;
    if (Kokkos::Profiling::profileLibraryLoaded()) {
      Kokkos::Profiling::beginParallelScan(
          label + " via thrust::inclusive_scan",
          Kokkos::Profiling::Experimental::device_id(ex), &kpID);
    }
```
before the call to `thrust::inclusize_scan` and
```C++
  if (Kokkos::Profiling::profileLibraryLoaded()) {
      Kokkos::Profiling::endParallelScan(kpID);
    }
```
after it. Similarly below.